src/components/util/timestamp-input/TimestampInput.tsx

```diff
@@ -15,28 +15,32 @@
  ***************************************************************************** */
 
 import * as React from 'react';
-import eventHttpApi from '../../api/event';
-import messageHttpApi from '../../api/message';
-import { formatTimestampValue } from '../../helpers/date';
-import { isEventAction } from '../../helpers/event';
-import { createBemElement } from '../../helpers/styleCreators';
-import { useDebouncedCallback } from '../../hooks';
-import { useOutsideClickListener } from '../../hooks/useOutsideClickListener';
-import { EventAction } from '../../models/EventAction';
-import { EventMessage } from '../../models/EventMessage';
-import { DateTimeMask } from '../../models/filter/FilterInputs';
-import { Timestamp } from '../../models/Timestamp';
-import KeyCodes from '../../util/KeyCodes';
-import '../../styles/timestamp-input.scss';
+import moment from 'moment';
+import eventHttpApi from '../../../api/event';
+import messageHttpApi from '../../../api/message';
+import { formatTimestampValue } from '../../../helpers/date';
+import { isEventAction } from '../../../helpers/event';
+import { createBemElement } from '../../../helpers/styleCreators';
+import { useDebouncedCallback } from '../../../hooks';
+import { useOutsideClickListener } from '../../../hooks/useOutsideClickListener';
+import { EventAction } from '../../../models/EventAction';
+import { EventMessage } from '../../../models/EventMessage';
+import { DateTimeMask } from '../../../models/filter/FilterInputs';
+import { Timestamp } from '../../../models/Timestamp';
+import KeyCodes from '../../../util/KeyCodes';
+import TimestampDialog from './TimestampDialog';
+import '../../../styles/timestamp-input.scss';
 
 const getTimestamp = (timestamp: Timestamp) => {
 	const ms = Math.floor(timestamp.nano / 1000000);
 	return +`${timestamp.epochSecond}${ms}`;
 };
 
-const dateFormatPattern = /[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])/g;
+const dateFormatPattern = /([0-9]{4})-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])/g;
 
 interface Props {
+	event: EventAction | null;
+	timestamp: number;
```

Comment:
Th-1401: 1) always display current minimap timestamp

I believe it expects you to update input timestamp(middle value of range) on range change.